Source/WebCore/dom/Element.cpp

```diff
@@ -2749,13 +2749,13 @@ bool Element::hasAutoTextDirectionState() const
 void Element::updateEffectiveTextDirection()
 {
     auto textDirectionState = elementTextDirectionState(*this);
-    updateEffectiveTextDirectionState(*this, textDirectionState);
+    updateEffectiveTextDirectionState(*this, textDirectionState, false);
```

Comment:
Can we add an enum class instead of using a boolean?

Reply:
I changed the fix since the old one seems to be incorrect.